src/argilla/client/feedback.py

```diff
@@ -484,7 +474,7 @@ def add_records(
             ...     ]
             ... )
             >>> dataset.records
-            [FeedbackRecord(fields={"text": "This is the first record", "label": "positive"}, responses=[{"user_id": None, "values": {"question-1": {"value": "This is the first answer"}, "question-2": {"value": 5}}}])]
+            [FeedbackRecord(fields={"text": "This is the first record", "label": "positive"}, responses=[ResponseSchema(user_id=None, values={"question-1": ValueSchema(value="This is the first answer"), "question-2": ValueSchema(value=5)})])]
         """
         if isinstance(records, list):
```

Comment:
I think `elif` statements would be better suited for the other cases

Reply:
Those are non-colliding if statements, but sure, I can make those `elif` instead, even though there's not a clear `else` condition, should we raise `ValueError`?